types/nodejs-captcha/index.d.ts

```diff
@@ -0,0 +1,68 @@
+// Type definitions for ssdeep.js 0.0
+// Project: https://github.com/ozgur-dogan/nodejs-captcha
+// Definitions by: JPBM135 <https://github.com/JPBM135>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/**
+ * Returns a captcha object
+ * @param p List of options, all of which are optional.
+ */
+declare function captcha(p: captchaOptions): captchaValue;
+interface captchaValue {
```

Comment:
While not required, the standard for names of interfaces, classes, types, etc. in TypeScript is to capitalize the first letter as well:

```suggestion
interface CaptchaValue {
```

Note that any references to `captchaValue` would have to be updated if this is changed